test/project_types/script/layers/application/push_script_test.rb

```diff
@@ -33,7 +34,14 @@
   let(:push_package_repository) { TestHelpers::FakePushPackageRepository.new }
   let(:extension_point_repository) { TestHelpers::FakeExtensionPointRepository.new }
   let(:script_project_repository) { TestHelpers::FakeScriptProjectRepository.new }
-  let(:task_runner) { stub(compiled_type: "wasm", metadata: metadata, library_version: library_version) }
+  let(:task_runner) do
+    stub(
+      compiled_type: "wasm",
+      metadata_file_location: metadata_file_location,
+      library_version: library_version,
+      respond_to?: true,
```

Comment:
This `respond_to?` stub isn't precise enough. I'd like for it to apply only when it has the `:build` argument.

I was looking at the [Mock docs](https://www.rubydoc.info/github/floehopper/mocha/Mocha/API#stub-instance_method) but I couldn't get it working by passing a block to the `stub(name, &block)`. The following didn't seem to work:

```
    stub('taskrunner') do
      stubs(:compiled_type).returns("wasm")
      ...more stubs
    end
```